util/provision-run/vars/vault

```diff
@@ -1,12 +0,0 @@
-$ANSIBLE_VAULT;1.1;AES256
-39343536636162396338386562386263636132343731643261366138386263373835616238303036
-3865336266326163646439393061636139363034643439620a653161313433336436323064643536
-34356236373462383239646165636661376130653435336462393066386636623239303831303963
-6635303730366662660a333239656161383536646233626165383564383936646337376639363864
-65356363643331336438373230626261393336353538613266663961396364366236356332633433
-64313533323237393261343432663137626339373162303939323361623666333566366162303664
-33346434613937346534363235663130346661663530333238333831386164643431353631303962
-64326632313065306237336339373361616130366139653530396534363063353535616530303964
-33626264633034383132623361353762363939633032663364633938323630316131626363376566
-36353337363438346537386335306535343765373565373430366338653033366336393665323534
-383839633333646530356434626530623530
```

Comment:
I think this is the rest of the code that's missing from the second file I reviewed? IDK...?

Reply:
It might be. Honestly, I have a hard time keeping track of these really long number

Reply:
😂🤣